.env.example

```diff
@@ -1,6 +1,7 @@
-POSTGRES_HOST=db
```

Comment:
Why change this? Would need to be `db` on the docker network, no?

Reply:
check, changed back.